src/main.cpp

```diff
@@ -1349,7 +1345,6 @@ void handleUploadLoop() {  // NOLINT(readability-function-cognitive-complexity)
     uploaderror = UploadError::fileUploadAborted;
     Update.end();
   }
-  delay(0);
```

Comment:
what would this have done?

Reply:
answer: [nothing](https://arduino.stackexchange.com/questions/86542/what-does-a-delay0-actually-do)